### ISSUE TITLE
Create initial branding resources

### ISSUE THREAD

**Issue by Nolski** [librecorps/tasks#11]
# Overview

I think given the amount of work we are doing, it would be incredibly useful to have at least some branding resources that we can use with out documents and communications. 

Ideally we would have a brand book

**Comment by justwheel** [comment]
This was blocked on budget. Ura quoted us a few thousand USD for what we were looking for, but it's not something in scope for us right now. I'm thinking we could use on the free logos from Ura's LogoBridge project for now though:

https://logobridge.co/

@Nolski @whenbellstoll What do you think of one of these two?

![Screenshot of two candidate LibreCorps logos from Ura's LogoBridge site](https://user-images.githubusercontent.com/4721034/77468084-78358a80-6de3-11ea-8bb9-ec22a62ab866.png "Screenshot of two candidate LibreCorps logos from Ura's LogoBridge site")

**Comment by Nolski** [comment]
Personally, as we already have a logo, I'd support staying with our current logo instead of switching to a stock one. Having a cohesive brand guide would be great for building recognition but I don't think switching logos would help us much.

I'd be happy to just close this ticket as it's out of scope unless there might be any students willing to take on a branding and design related project

**Comment by justwheel** [comment]
@Nolski That's valid feedback, but my motivation is because I don't have a SVG or original source document for the current logo. Do you have it?

**Comment by Nolski** [comment]
No but perhaps sj has it. If not, you can probably run it through inkscape and generate an svg

**Comment by justwheel** [comment]
@Nolski I'm not sure if @itprofjacobs has the original LibreCorps logo source, but I tagged him here in case he does.

I tried tracing a bitmap in Inkscape, but this is the best I got for now:

![Deformed LibreCorps logo](https://user-images.githubusercontent.com/4721034/78511191-c09c6300-7768-11ea-9fee-539592260cb4.png)
